src/txn/txn_log.c

```diff
@@ -660,6 +660,13 @@ __txn_printlog(WT_SESSION_IMPL *session, WT_ITEM *rawrec, WT_LSN *lsnp, WT_LSN *
     /* First, peek at the log record type. */
     WT_RET(__wt_logrec_read(session, &p, end, &rectype));
 
+    if (F_ISSET(args, WT_TXN_PRINTLOG_MSG)) {
```

Comment:
I'd add a comment here about how this flag is used. It is not obvious to reason about without context when looking at printing out log message records further down.

Also, I think it would be useful to know the LSN for the messages so that one can easily find the log file and see the records and activity around that time.

Reply:
I added a comment in 59290b1b1.

I don't want to add the LSN (there are timestamps), but if we ever offer a way to print out a range of log records, I think it's easier to make that argument.